config/settings.py

```diff
@@ -192,3 +192,28 @@
 
 OIDC_AFTER_USERLOGIN_HOOK = "sso.hooks.do_autologin_after_successful_login"
 OIDC_USERINFO = "config.oidc_provider_settings.userinfo"
+
+LOG_LEVEL = "DEBUG" if DEBUG else "INFO"
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "handlers": {
+        "infoconsole": {
+            "level": LOG_LEVEL,
+            "class": "logging.StreamHandler",
+            "formatter": "verbose",
+        },
+    },
+    "loggers": {
+        "": {
```

Comment:
La clé vide signifie : LOG ALL THE THINGS
Sinon on pourrait mettre un namespace (par exemple tout ce qui concerne notre SSO dans un fichier à part).